src/abt.sig

```diff
@@ -49,7 +110,15 @@ sig
   structure BFunctor : FUNCTOR
     where type 'a t = 'a bview
 
-  (* construct an abt from a view by checking it against a sort *)
+  (* Note that if you're used to CMU's abt library or Jon's previous one,
+   *  - into ===> check
+   *  - out  ===> infer
+   *)
+
+  (* construct an abt from a view by checking it against a sort. Note
+   * that in the presence of sorts this operation CAN FAIL. This is
+   * different than you may expect if you've used other abt libraries.
+   *)
```

Comment:
This operation can also fail in non-sortful abt libs, in case you got the arity wrong or something.